src/besselk.jl

```diff
@@ -431,3 +434,68 @@ end
 besselk_power_series_cutoff(nu, x::Float64) = x < 2.0 || nu > 1.6x - 1.0
 besselk_power_series_cutoff(nu, x::Float32) = x < 10.0f0 || nu > 1.65f0*x - 8.0f0
 
+
+"""
+  besselk_asymptoticexp(v, x, order)
+
+Computes the asymptotic expansion of K_ν w.r.t. argument. Accurate for large x, and faster than uniform asymptotic expansion for small to small-ish orders. The default order of the expansion is 10.
+"""
+function besselk_asymptoticexp(v, x::T) where T 
+  _besselk_asymptoticexp(v, float(x), 10)
+end
+besselk_asexp_cutoff(nu, x) = (nu < 20.0) && (x > 25.0)
+
+# Compute the function for (_v, _v+1), where _v = v-floor(v), and then do
+# forward recursion. If the argument is really large, this should be faster than
+# the uniform asymptotic expansion and plenty accurate.
+function _besselk_asymptoticexp(v, x::T, order) where T
```

Comment:
Could you fix the indentation spacing here? Four spaces for each indent.

Reply:
Totally, will do in next commit.

Reply:
Fixed!